### DISCUSSION TITLE
Interpolate points in a mesh from nifti DVF image volume

### DISCUSSION THREAD

**Discussion by isurusuranga** [pyvista/pyvista#1763] [Q&A]
Hi,

I have generated a volumetric mesh and saved it as a vtkUnstrcturedGrid (i.e. a .vtk file). Here I extracted a volumetric mesh from a nifti image which has a dimension of (203, 127, 161). I have then applied a non-rigid deformation to this nifti file and saved the deformation vector field as another nifti file. The shape of this DVF nifti mage is (203, 127, 161, 1, 3). Here the deformation field is the final position of the voxel after it has been transformed, i.e. it is the initial position of the voxel + the displacement of the voxel.

Now I need to distort the volumetric mesh based on this deformation vector field. How can I accomplish this using pyvista ? Could you please give an example ? It is possible to attach a sample nifti DVF and a voluemtric mesh for your further reference.

I highly appreciate your willingness to help me out on this issue.

**Comment by RichardScottOZ** [comment]
A good idea...and maybe what they look like...and in code?
